hysds/dataset_ingest.py

```diff
@@ -604,7 +603,7 @@ def ingest(objectid, dsets_file, grq_update_url, dataset_processed_queue,
         'ipath': ipath,
         'system_version': version,
         'dataset_level': level,
-        'dataset_type': dtype,
```

Comment:
@DustinKLo: let's preserve `dataset_type` to be what is specified in the `datasets.json` for the respective recognized dataset e.g. https://github.com/hysds/hysds/blob/develop/configs/datasets/datasets.json#L7.